pulumi/aws/observability/otel-objects/otel-collector.yaml.full

```diff
@@ -0,0 +1,65 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: simplest
+  namespace: observability
+spec:
+  config: |
+    extensions:
+      health_check:
+      pprof:
+        endpoint: 0.0.0.0:1777
+      zpages:
+        endpoint: 0.0.0.0:55679
+    
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:9978
+          http:
+            endpoint: 0.0.0.0:9979
+      opencensus:
+      jaeger:
+        protocols:
+          grpc:
+          thrift_binary:
+          thrift_compact:
+          thrift_http:
+      zipkin:
+    
+      # Collect own metrics
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'otel-collector'
+              scrape_interval: 120s
+              static_configs:
+                - targets: [ '0.0.0.0:8080']
+              metrics_path: '/z/prometheus'
+    
+    processors:
+      batch:
+    
+    exporters:
+      prometheus:
+        endpoint: "0.0.0.0:8889"
+    
+      logging:
+        logLevel: debug
+    
+      jaeger:
+        endpoint: "0.0.0.0:14250"
+     
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp, opencensus, jaeger, zipkin]
```

Comment:
Do we need the jaeger, opencensus and zipkin receivers?

Reply:
I'm leaving all of those in for now, since that is the default configuration for OTEL on their webpage showing how to setup the simplest collector; my thought is that this mirrors what you would get if you just followed their instructions. Everything is configured - that is, they work fine, it's just opening ports to listen or allow connections. Once we move further down the configuration process I'd like to move this aside and create a new config with just the pieces we need for our chosen solution.